frontend/dockerfile/dockerfile2llb/convert_test.go

```diff
@@ -230,3 +230,43 @@ RUN echo bar
 	assert.Equal(t, []digest.Digest{"sha256:2e112031b4b923a873c8b3d685d48037e4d5ccd967b658743d93a6e56c3064b9"}, baseImg.RootFS.DiffIDs)
 	assert.Equal(t, "2024-01-17 21:49:12 +0000 UTC", baseImg.Created.String())
 }
+
+func TestGetSecretsRegex(t *testing.T) {
+	t.Parallel()
+	deny, allow := getSecretsRegex()
+
+	testCases := []struct {
+		name     string
+		key      string
+		isSecret bool
+	}{
+		// Positive matches
```

Comment:
None of these tests seem to actually check for the change of behavior (handling secrets in the middle of string).